app/views/root/index.html.erb

```diff
@@ -0,0 +1 @@
+<div id="root"></div>
```

Comment:
- [ ] It would be better if we install the packages before using them in our code, because of not install `react-routr-dom`, `style-loader`, `react-redux`, `@reduxjs-toolkit` and `redux-logger` we have errors running the app. Kindly check the `package.json` file to see if these files are present or not and install them accordingly
![Screenshot from 2023-08-15 12-39-32](https://github.com/DJ-MrJay/hello-rails-react/assets/46388113/4d8d2838-1bf6-4c6d-8885-31376caa2497)